<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMainWindow</class>
 <widget class="QMainWindow" name="TestMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Adresseingabe" name="widgetAdresseingabe" native="true">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1144</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionOeffnen"/>
    <addaction name="actionSpeichern"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuAdresse">
    <property name="title">
     <string>Adresse</string>
    </property>
    <addaction name="actionHinzufuegen"/>
    <addaction name="actionLoeschen"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuAdresse"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOeffnen">
   <property name="text">
    <string>Öffnen</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionHinzufuegen">
   <property name="text">
    <string>Hinzufügen</string>
   </property>
  </action>
  <action name="actionLoeschen">
   <property name="text">
    <string>Löschen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Adresseingabe</class>
   <extends>QWidget</extends>
   <header>Adresseingabe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionBeenden</sender>
   <signal>triggered()</signal>
   <receiver>TestMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
